File: C++/Source Code C++ Tham khảo/KieuDuLieu,BienVaToanTu/Bai14_XepLoaiHocSinh.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    double a , b , c , d ; cin >> a >> b >> c >> d;
    double tb = a + b + c*2 + d * 3;
    tb /= 7;
    if(tb >= 8) cout << "GIOI";
    else if(tb >= 6.5) cout << "KHA";
    else if(tb >= 5) cout << "TRUNG BINH";
    else cout << "YEU";
}

File: Python/Source Code Python Contest/Contest_1_KieuDuLieu_ToanTu_ReNhanh/Bai14_XepLoaiHocSinh.py
score_1, score_2, score_3, score_4 = map(float, input().split())
average_score = (score_1 + score_2 + score_3 * 2 + score_4 * 3) / 7
if average_score >= 8:
    print('GIOI')
elif average_score < 8 and average_score >= 6.5: # cach 2 average_score >= 6.5
    print('KHA')
elif average_score < 6.5 and average_score >= 5: # cach 2 average_score >= 5
    print('TRUNG BINH')
else:
    print('YEU')